packages/functions/src/getFullTestFeedback.ts

```diff
@@ -0,0 +1,67 @@
+import { APIGatewayProxyHandlerV2 } from 'aws-lambda';
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb';
+import {
+  DynamoDBDocumentClient,
+  GetCommand,
+  PutCommand,
+  UpdateCommand,
+} from '@aws-sdk/lib-dynamodb';
+import { Table } from 'sst/node/table';
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import { FullTestItem } from './utilities/fullTestUtilities';
+
+const client = new DynamoDBClient();
+const dynamoDb = DynamoDBDocumentClient.from(client);
+
+export const main = async (
+  event: APIGatewayProxyEvent,
+): Promise<APIGatewayProxyResult> => {
+  const testSK = event.pathParameters?.SK;
+
```

Comment:
From what I can see, the testSK might be null.  Why not add a check to unsure a sort key was provided and inform the front end in case of failure to provide the required parameter.  (Better than failing dynamoDB call and getting cryptic 500 error code)